### ISSUE TITLE
Package request: lftp 

### ISSUE THREAD

**Issue by sebastian-kreis** [nanobox-io/nanobox-pkgsrc-base#66]
http://lftp.tech/

**Comment by danhunsaker** [comment]
Note: This is a file transfer client, not a server.

(cc: @notxarb)

**Comment by sebastian-kreis** [comment]
Yes. It has a useful mirror function to sync folders. Either local->remote, remote->local or remote->remote. My use case would be in a cron job to keep folders in sync.
